src/Console/Command/DispatchCommand.php

```diff
@@ -373,7 +373,10 @@ private function dispatchFiles(Package $package): void
             }
             // If the previous branch is not merged into the current one, do nothing.
             if ($previousBranch && $this->githubClient->repos()->commits()->compare(
-                    static::GITHUB_GROUP, $repositoryName, $currentBranch, $previousBranch
+                static::GITHUB_GROUP,
+                $repositoryName,
+                $currentBranch,
+                $previousBranch
                 )['ahead_by']) {
```

Comment:
There is an indention issue here.

Reply:
If I am correct the indention is not checked by this rule automatically.

Anyway I like the result but we should check this after the dev kit updates and FlintCI fix PRs

Reply:
Yes, you're correct IIRC, we will have to check. Luckily, people seldom violate that rule.

Reply:
In any case, there is an indentation issue here. No? :-)

Reply:
@phansys can you please fix this?

Reply:
The proposed fix doesn't look right to me (https://flintci.io/repositories/50/analyses/24032):
```diff
@@ -373,11 +373,11 @@ final class DispatchCommand extends AbstractNeedApplyCommand
             }
             // If the previous branch is not merged into the current one, do nothing.
             if ($previousBranch && $this->githubClient->repos()->commits()->compare(
-                    static::GITHUB_GROUP,
-                    $repositoryName,
-                    $currentBranch,
-                    $previousBranch
-                )['ahead_by']
+                static::GITHUB_GROUP,
+                $repositoryName,
+                $currentBranch,
+                $previousBranch
+            )['ahead_by']
             ) {
                 continue;
             }
```
Maybe we should squash this call into a single line.

Reply:
Can you post an issue to the PHP-CS-Fixer guys? Maybe it gets fixed by another release? 

@Soullivaneuh which version is FlintCI using?